cloud_guardian/iam_model/graph/main.py

```diff
@@ -75,18 +98,84 @@
     group = GroupFactory.from_dict(group_data)
     for user_data in group_data["Users"]:
         user = UserFactory._instances[user_data["UserArn"]]
-        is_part_of_relationship = IsPartOf(user, group)
-        graph.add_relationship(is_part_of_relationship)
+        graph.add_relationship(IsPartOf(user, group))
 
 
-# TODO: Parse the rest of the relationships (see relationships.py)
+# Can Assume Role relationships
+for role_data in data["roles.json"]["Roles"]:
+    role = RoleFactory.from_dict(role_data)
+    for statement in role_data["AssumeRolePolicyDocument"]["Statement"]:
+        if statement["Effect"] != "Allow":
+            continue
+
+        # TODO: implement a mechanism to ensure that the list of users is complete
+        try:
+            user = UserFactory._instances[statement["Principal"]["AWS"]]
+        except KeyError as e:
+            logger.error(f"user does not exist {e}")
+
+        graph.add_relationship(CanAssumeRole(user, role))
+        logger.info(
+            f"added relationship between {user.user_name} and role {role.role_name}: can assume role"
+        )
 
-# roles.json contain all you need to make the "CanAssumeRole" relationships
+# Users permissions
+for user_data in data["users.json"]["Users"]:
+    # TODO: implement a mechanism to ensure that the list of users is complete
+    try:
+        user = UserFactory._instances[user_data["Arn"]]
+    except KeyError as e:
+        logger.error(f"user does not exist {e}")
+
+    for policy_data in user_data["AttachedPolicies"]:
+        arn = policy_data["PolicyArn"]
+        policy = extract_policy_from_ARN(arn)
+
+        permission = PermissionFactory.get_or_create(
+            action=ActionFactory.get_or_create(arn),
+            effect=Effect(
+                Effect.ALLOW
+            ),  # NOTE: do we assume all permissions ALLOW by default?
+            conditions=[],
+        )
+
+        # FIXME: how do we define the target here? (I am temporarily using `None`)
```

Comment:
target is the "Resource" field in the "identty_policies.json" 
and the "ResourceArn" field in the "resources_policies.json"
TODO: can change both to be "ResourceArn" as well for consistency

Reply:
Done.
I did not change both to `ResourceArn` as `identities_policies.json` contains both, which is confusing (I let you decide).